pkg/vpcmodel/output.go

```diff
@@ -237,13 +240,19 @@ func WriteToFile(content, fileName string) (string, error) {
 	return content, nil
 }
 
-// getAsteriskDetails returns the info message about how non stateful conns are marked in the output, when relevant
-func getAsteriskDetails(uc OutputUseCase, hasStatelessConn bool, outFormat OutFormat) string {
+// getAsteriskDetails returns:
+// 1. the info message about how non stateful conns are marked in the output, when relevant
+// 2. the info message about how overAproximated conns are marked in the output, when relevant
+func getAsteriskDetails(uc OutputUseCase, hasStatelessConn, hasOverApproximatedConn bool, outFormat OutFormat) string {
+	res := ""
```

Comment:
Better add:
```

if  uc == SingleSubnet || (outFormat == Text && outFormat == MD && outFormat == Debug) {
  return res
}
```

Reply:
I do not follw

Reply:
first check the repeating condition in which an empty string is returned